### ISSUE TITLE
Should we support running different hyperparameter settings for a workload in parallel?

### ISSUE THREAD

**Issue by runame** [mlcommons/algorithmic-efficiency#225]
## Description

Currently, we run all hyperparameter setting defined in the search space sequentially for each workload ([for-loop here](https://github.com/mlcommons/algorithmic-efficiency/blob/main/submission_runner.py#L446)). Users might want to parallize not only running their submission on the different workloads, but also running with different hyperparameter settings. Should we support that?

**Comment by znado** [comment]
Yeah I think this could be useful, we could add a string flag like `--trial_range=x,y` (that defaults to an empty string and then does the current behavior), but as long as we generate the search space with the same seed we should be able to subslice the hparam list with `[x:y]`.

However, I think this is a super low priority compared to getting all the workloads finalized :)

**Comment by runame** [comment]
Addressed by #503.
